code/controllers/subsystems/ticker.dm

```diff
@@ -436,9 +436,9 @@ var/datum/controller/subsystem/ticker/SSticker
 	sleep(1)	// Sleep so the MC has a chance to update its init time.
 	if(!login_music)
 		if(SSatlas.current_sector && SSatlas.current_sector.lobby_tracks)
-			login_music = pick(SSatlas.current_sector.lobby_tracks)
+			login_music = SSatlas.current_sector.lobby_tracks
```

Comment:
You should shuffle the list around, otherwise it would start playing the same one every time?

Reply:
intentional - i want yuki satellites to play first

ideally it would play a selected song first (if set) then randomize, but i'm not smart enough for that